dotnet/src/Connectors/Connectors.Google/Core/Gemini/Models/GeminiRequest.cs

```diff
@@ -109,6 +116,20 @@ private static GeminiContent CreateGeminiContentFromChatMessage(ChatMessageConte
         };
     }
 
+    private static GeminiContent? CreateSystemMessages(ChatHistory chatHistory)
```

Comment:
Should we update our Gemini samples and include system messages?

Reply:
Good idea. I also think about adding new sample to Agents which utilises openai and gemini at the same time, because now it is possible, what do you think?

Reply:
I will update system message sample here but sample as diffrent PR.